app/views/tpi/publications/_promoted.html.erb

```diff
@@ -1,70 +1,30 @@
 <div class="promoted-publications__publications">
-  <div class="columns">
-    <div class="column is-one-third publication">
-      <img src="<%= asset_path 'mocks/tpi_publications_01.jpg' %>" class="publication__avatar">
-
-      <div class="publication__tags">
-        <span class="tag">Airlines</span>
-        <span class="tag">Sector reports</span>
-      </div>
-
-      <div class="publication__content">
-        <p class="updated_at">Mar 5, 2019</p>
-
-        <div class="content__title">
-          <a class="link is-strong">TPI State of Transition Report 2019</a>
-        </div>
-
-        <div class="content__message">
-          <p>
-            By Simon Dietz, Rhoda Byrne, Valentin Jahn, Michal Nachmany, Jolien Noels and Rory Sullivan
-          </p>
-        </div>
-      </div>
-    </div>
-
-    <div class="column is-one-third publication">
-      <img src="<%= asset_path 'mocks/tpi_publications_02.jpg' %>" class="publication__avatar">
-
-      <div class="publication__tags">
-        <span class="tag">Airlines</span>
-      </div>
-
-      <div class="publication__content">
-        <p class="updated_at">Mar 5, 2019</p>
-
-        <div class="content__title">
-          <a class="link is-strong">TPI research on world’s top airlines generates considerable global media coverage</a>
-        </div>
-        <div class="content__message">
-          <p>
-            On 5 March, TPI released new research on the world’s top 20 publicly-listed airline companies. Based on analysis by […]
-          </p>
-        </div>
-      </div>
-    </div>
-
-    <div class="column is-one-third publication">
-      <img src="<%= asset_path 'mocks/tpi_publications_03.jpg' %>" class="publication__avatar">
-
-      <div class="publication__tags">
-        <span class="tag">Airlines</span>
-        <span class="tag">Sector reports</span>
-      </div>
-
-      <div class="publication__content">
-        <p class="updated_at">Mar 5, 2019</p>
-
-        <div class="content__title">
-          <a class="link is-strong">New research shows only two large oil & gas companies have long-term, low-carbon ambitions</a>
-        </div>
-
-        <div class="content__message">
-          <p>
-            Innovative new research assesses the carbon performance of the world’s ten largest public oil and gas companies, taking into […]
-          </p>
+  <div class="publications__grid">
```

Comment:
IMHO this will make it harder for mobile.

Reply:
what is your concern? I just pushed the responsive version; it's only changing the `grid-template-columns` property

Reply:
It was easier with bulma's flex grid. Don't see the point of overriding everything is all.

Reply:
we had code for three columns only without wrapping up to the next rows; CSS grid handles that use case perfectly and it's very responsive. If we decide that instead of three columns, we want to have 4,5,6, you name it, you need to change one character; `grid-template-columns: repeat(3, 1fr);` 3 to 4,5,6, etc.
I don't mind using bulma, it's not for every use case though. Using grid here for much easier; otherwise with Bulma I'd probably have to change html structure.

Reply:
I understand that for desktop, but for mobile it's just one each row. Not so convenient. But i'd like to be proved wrong.

Reply:
I thought I just did 😅
I don't understand what's inconvenient here. for having responsive implementation you don't need to change anything in the html structure. Everything you need to make it responsive for desktop is in the `@media tablet-portrait` block, as I tried to make it mobile-first. If we wanted to have even more breakpoints, desktop large, smaller mobile etc. we'd also need to change just this one CSS property. To me, it's very convenient - everything is in the CSS.

Reply:
Take a look here: https://github.com/Vizzuality/laws_and_pathways/pull/158/files#diff-a314848eddcc2930479906ab6a4efa76R158
(linking it here as I added it after we started this conversation)

Reply:
Ah! Right! I've seen it!
Proved wrong.
P.S.: you can use an if/else clause with bulma's breakpoint variables instead of adding media queries

Reply:
ah, true! will change that, thanks